test/optimization/test_slsqp.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+""" Test SLSQP Optimizer """
+
+import logging
+import unittest
+
+from test.optimization.optimization_test_case import QiskitOptimizationTestCase
+
+import numpy as np
+from qiskit.optimization.algorithms import SlsqpOptimizer
+from qiskit.optimization.problems import QuadraticProgram
+
+
+logger = logging.getLogger(__name__)
```

Comment:
logger is not needed/used for the unit test I think.

Reply:
Removed